packages/yoga/src/Popover/web/Popover.jsx

```diff
@@ -64,9 +69,9 @@ const Popover = React.forwardRef(
             $zIndex={zIndex}
           >
             {!!title && (
-              <Text.Small mb="xxxsmall" fw="medium" color="white">
+              <PopoverTitle mb="xxxsmall" color="white">
```

Comment:
We can probably use the `Text.Body2 bold` to define the text instead of updating the font weight locally in the Popover component.

WDYT?

```suggestion
              <Text.Body2 bold mb="xxxsmall" color="white">
```

Reply:
![image](https://github.com/gympass/yoga/assets/5889973/67395813-faff-4369-ad1d-b3c9d8f21d8a)